hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/storage/StorageClustersProperties.java

```diff
@@ -1,18 +1,28 @@
-package pl.allegro.tech.hermes.management.config;
+package pl.allegro.tech.hermes.management.config.storage;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import pl.allegro.tech.hermes.management.infrastructure.dc.DcNameSource;
+
+import java.util.ArrayList;
+import java.util.List;
 
 @ConfigurationProperties(prefix = "storage")
-public class StorageProperties {
+public class StorageClustersProperties {
 
     private String pathPrefix = "/hermes";
-    private String connectionString = "localhost:2181";
-    private int sessionTimeout = 10000;
-    private int connectTimeout = 1000;
     private int retryTimes = 3;
     private int retrySleep = 1000;
     private int sharedCountersExpiration = 72;
+    private DcNameSource dcNameSource;
```

Comment:
Same here, I'd rename `DcNameSource` to `DatacenterNameSource` in class name as well as the property name.